src/openforms/scss/admin/themes/_dark.scss

```diff
@@ -29,6 +29,7 @@
   --default-button-hover-bg: #888;
 
   --form-input-bg: #232222;
+  --form-input-bg-darker: #1a1818;
```

Comment:
As the other variable names matches the component state (e.g. `list-item-active-bg`), maybe this could be `form-input-bg-readonly`?